pkg/skaffold/util/util.go

```diff
@@ -106,6 +106,13 @@ func ExpandPathsGlob(workingDir string, paths []string) ([]string, error) {
 	return ret, nil
 }
 
+// HasMeta reports whether path contains any of the magic characters
+// recognized by filepath.Match.
+// This is a copy of filepath/match.go's hasMeta
```

Comment:
the original implementation also includes the filepath separator. why not just use that?

Reply:
https://golang.org/pkg/path/filepath/#Match?